client/router/index.ts

```diff
@@ -21,6 +24,18 @@ const router = createRouter({
       component: SettingView,
       meta: { requiresAuth: true },
     },
+    {
+      path: "/news_",
```

Comment:
What's up with the `news_` and not just `news`?

Reply:
Good question, I also noticed this. I think it should be changed to just `news` for consistency